cmd/web/main.go

```diff
@@ -142,7 +142,7 @@ func main() {
 	g.GET("/mr-stack/merged", app.GetMergeRequestMergedStack)
 	g.GET("/mr-stack/closed", app.GetMergeRequestClosedStack)
 
-	g.GET("/mr/:mrid", app.GetMergeRequestDetails)
+	// g.GET("/mr/:mrid", app.GetMergeRequestDetails)
```

Comment:
Remove the handler also and templates that are connected to it. 

Please don't comment out the code, delete it.